nextcloud/android
# Show slideshow as new activity

Superseeds #4544

To test:
- create a slideshow in Collabora
- press slideshow button
- see slideshow in new activity, with left/right to go through slides

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>
